os/drivers/lcd/mipi_lcd.c

```diff
@@ -44,6 +44,11 @@
 extern const uint8_t lcd_logo_raw_data[]; // Buffer containing only logo
 static uint8_t *lcd_init_fullscreen_image = NULL; // Buffer containing full screen data with logo on specific position
 
+enum {
+	CMD = 0,
+	VIDEO = 1
```

Comment:
`CMD` and `VIDEO` are too common names.
Let me suggest 
```
enum mipi_mode_e {
    MIPI_CMD_MODE = 0,
    MIPI_VIDEO_MODE =1
};
```

And please find Taejun's comment below.